src/lib/constants.js

```diff
@@ -0,0 +1,73 @@
+const ethers = require('ethers');
+
+module.exports = {
+  CONTRACTS_ADDRESSES: {
```

Comment:
Let's have a separate file for the contract addresses

Reply:
This will be removed once, we can use contractFactory to get the abis and addresses...